src/updateData.js

```diff
@@ -0,0 +1,113 @@
+import { getForengeitScale } from './changeTempScale';
+
+export const currentWeatherTemp = document.querySelector('.weather--temp');
+export const feelsLikeTemp = document.querySelector('.feels--temp');
+export const forecastTempText = document.querySelectorAll('.future--temp');
+const latitude = document.querySelector('.latitude');
+const longitude = document.querySelector('.longitude');
+const cityAndCountry = document.querySelector('.location--text');
+const forecastWeatherImg = document.querySelectorAll('.future--img img');
+const forecastDayText = document.querySelectorAll('.future--day--name');
+const feelsLikeText = document.querySelector('.feels--text');
+const windText = document.querySelector('.wind');
+const humidityText = document.querySelector('.humidity');
+const currentWeatherImg = document.querySelector('.weather--temp--img img');
+const descriptionWeatherText = document.querySelector('.description');
+const searchPlace = document.querySelector('.input--block');
+const searchBtn = document.querySelector('.search--btn');
+const milisecond = 1000;
+let interval;
+
+export const updateTime = async (placeData, language) => {
+  clearInterval(interval);
+  const { coordinates } = placeData;
+  const timeZone = await getTimeZone(...coordinates);
+  interval = setInterval(getTime.bind(null, timeZone, language), 1000);
+};
+
+export const updateAppData = async (placeData, language) => {
+  const latitudeText = language === 'ru' ? 'Широта' : 'latitude';
+  const longitudeText = language === 'ru' ? 'Долгота' : 'longitude';
+  const searchText = language === 'ru' ? 'Поиск' : 'Search';
+  cityAndCountry.innerHTML = `${placeData.city ? placeData.city : ''} ${
+    placeData.country
+  }`;
+  const [latitudeValue, longitudeValue] = placeData.coordinates;
+  latitude.innerHTML = `${latitudeText}: ${String(
+    latitudeValue.toFixed(2),
+  ).replace(/[.]/g, '°')}'`;
+  longitude.innerHTML = `${longitudeText}: ${String(
+    longitudeValue.toFixed(2),
+  ).replace(/[.]/g, '°')}'`;
+  searchBtn.textContent = searchText;
+  searchPlace.placeholder = searchText;
+  return placeData.coordinates;
+};
+
+export const updateForecastWeather = async (placeData, language) => {
+  let dayNameIndex = 0;
+  let dayTempIndex = 0;
+  let dayIconIndex = 0;
+  const { coordinates } = placeData;
+  const weatherData = await getWeather(language, coordinates);
+  const weather = weatherData.forecast;
+  const arrayDayDate = [
+    weather.firstDayData.dt * milisecond,
+    weather.secondDayData.dt * milisecond,
+    weather.thirdDayData.dt * milisecond,
+  ];
+  const arrayDayTemp = [
+    weather.firstDayData.temp.day.toFixed(),
+    weather.secondDayData.temp.day.toFixed(),
+    weather.thirdDayData.temp.day.toFixed(),
+  ];
+  const arrayDayIcon = [
+    weather.firstDayData.weather[0].icon,
+    weather.secondDayData.weather[0].icon,
+    weather.thirdDayData.weather[0].icon,
+  ];
+  forecastTempText.forEach((tempText) => {
```

Comment:
to long function, you can split it to separate functions that updates tempreture text, than day text and so on...